source/dbo/Stored Procedures/Data Migration/uspConfirmationLineList.sql

```diff
@@ -18,10 +18,15 @@ AS
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_Hospital] tbh ON tbh.HospitalID = mn.NtbsHospitalId
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_PHEC] tbsp ON tbsp.TB_Service_Code = tbh.TB_Service_Code
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[PHEC] p ON p.PHEC_Code = tbsp.PHEC_Code
+		LEFT JOIN vwNotificationYear ny ON ny.NotificationYear = YEAR(mn.NotificationDate)
 	WHERE 
 		mn.GroupId IS NOT NULL
 		AND p.PHEC_Name = @Region
-		AND mn.NotificationDate >= '2017-01-01'),
+		AND ny.Id >= -3
+		AND NOT EXISTS
+			(SELECT LegacyId
```

Comment:
Now, I appreciate this SHOULD work but we've never relied on the ImportedNotifications table. I would go to NTBS to see if the value is there, rather than relying on it, but happy to discuss.

Reply:
If it's unreliable then changing it seems like the better option. This table is also used in search to filter out already imported notifications, maybe we should look at removing it altogether and just going to the notification table instead?

Reply:
Yes, this is true - either it's reliable or it isn't. And it was put in originally as a safety measure to be sure we didn't offer the user a legacy notification to import that had actually already been imported, this is the ticket: https://airelogic-nis.atlassian.net/browse/NTBS-1440  so on that basis we SHOULD be able to use it.

Reply:
I have commit a change that uses the NTBS notifications table instead. It seems like it's the safest option, as the app showing a legacy notification seems a little less risky than a reporting showing false data. I did the change in one commit, so if we decide to use the imported table then reverting shouldn't be too bad.